var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -202,6 +202,7 @@ class Sundials(CMakePackage, CudaPackage, ROCmPackage):
     patch('FindPackageMultipass.cmake.patch', when='@5.0.0')
     patch('5.5.0-xsdk-patches.patch', when='@5.5.0')
     patch('0001-add-missing-README-to-examples-cvode-hip.patch', when='@5.6.0:5.7.0')
+    patch('remove-links-to-OpenMP-vector.patch', when='@5.5.0:5.7.0')
```

Comment:
Can you add a comment on top of the patch directive that contain provenance of the patch and briefly explain what the patch does?